app/src/main/java/com/iitp/njack/iitp_connect/data/database/DatabaseContentProvider.java

```diff
@@ -103,7 +105,7 @@ public int delete(@NonNull Uri uri, @Nullable String selection, @Nullable String
         switch (match) {
             case CONTESTS:
                 itemsDeleted = db.delete(DatabaseContract.ContestEntry.TABLE_NAME_CONTESTS, null, null);
-                Log.v("ContentProvider", "All the previous contests deleted i.e. " + itemsDeleted + " contests");
+                Timber.v("ContentProvider", "All the previous contests deleted i.e. " + itemsDeleted + " contests");
```

Comment:
In order to avoid the ```TimberArgCount```, create a String above it with the concatenated values and use that.

Reply:
Oh, sorry , omw!

Reply:
i already tried :

String concatMsg = "All the previous contests deleted i.e. " + itemsDeleted + " contests";
Timber.v("ContentProvider",concatMsg);

but i'm still getting the suppress suggestion

Reply:
```Timber.v(concatMsg);```

Reply:
The TimberArgCount suggestion was because you're passing two arguments in it.